src/Tokenizer/TokensAnalyzer.php

```diff
@@ -580,15 +580,27 @@ private function findClassyElements(Tokens $tokens, $start)
             }
 
             if (0 === $bracesLevel && $token->isGivenKind(T_VARIABLE)) {
-                $elements[$index] = ['token' => $token, 'type' => 'property'];
+                $elements[$index] = [
+                    'token' => $token,
+                    'type' => 'property',
+                    'class' => $inClass,
```

Comment:
what is here? boolean ?

Reply:
`int`, index in the Tokens collection of the `classy` token

Reply:
please add `index` or sth...

Reply:
done